site_scons/buildutils.py

```diff
@@ -407,61 +500,68 @@ def compareCsvFiles(env, file1, file2):
     files are dissimilar. Lines containing non-numeric data are
     automatically ignored.
     """
-    try:
-        import numpy as np
-    except ImportError:
-        print('WARNING: skipping .csv diff because numpy is not installed')
-        return 0
+    if not np:
+        logger.warning(
```

Comment:
does this fit on one line with 88 characters? This is one of the `black` formatting choices I don't understand.

Reply:
Yes, this was this way from a previous point in the history when it was called `build_logger` rather than `logger`, and the `build_` tipped it over 88 characters. I'm not actually using Black to format the file 😄